.vscode/settings.json

```
@@ -20,5 +21,10 @@
       "url": "./.vscode/package.json-schema.json"
     }
   ],
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "editor.quickSuggestions": {
+    "strings": true
+  },
+  "tailwindCSS.classFunctions": ["tw", "tw`", "twClassName"],
```

Change this to classnames until we add twMerge
```suggestion
  "tailwindCSS.classFunctions": ["classnames"],
```